### ISSUE TITLE
Overriding too much

### ISSUE THREAD

**Issue by flaviocopes** [OleVik/grav-plugin-adminidenticons#1]
https://github.com/OleVik/grav-plugin-adminidenticons/blob/master/admin/templates/partials/nav.html.twig

caused me a "whaaat" moment, I had the plugin installed and any edit to Admin's original Twig was not working. 

I understand the use case of changing the avatar, or even removing it.
I'd propose in `nav.html.twig`:

```twig
        {% include 'partials/nav-user-details.html.twig' %}
```

`nav-user-details.html.twig`:
```twig
        <div id="admin-user-details">
            <a href="{{ base_url_relative }}/user/{{ admin.user.username|e }}">
                {% include 'partials/nav-user-avatar.html.twig' %}

                <div class="admin-user-names">
                    <h4>{{ admin.user.fullname|e }}</h4>
                    <h5>{{ admin.user.title|e }}</h5>
                </div>
            </a>
        </div>
```

and `nav-user-avatar.html.twig`:
```twig
<img src="https://www.gravatar.com/avatar/{{ admin.user.email|md5 }}?s=47" />
```

what do you think? So you can simply override `partials/nav-user-avatar.html.twig`, while other uses might even remove the avatar, or even the name.

**Comment by OleVik** [comment]
The solution I made was very much trying to find a workable way to implement the avatar, so I'm all for any change that will make it easier to plug it in that is less obstructive. Ping me when [#989](https://github.com/getgrav/grav-plugin-admin/pull/989) is merged, so I can implement the less-destructive template override.

Another, related, issue is [range.html.twig](https://github.com/OleVik/grav-plugin-adminidenticons/blob/master/admin/templates/forms/fields/range/range.html.twig), which I would suggest as an improvement for the Admin-plugin, but the current solution is not as robust or aesthetically optimal as it should be. The range-slider, as it currently stands in the Admin-plugin, provides a very low level of specificity for the selected value, thus I added a simple `<output>`-tag to show it in. But it is rather obstructive as it too overrides the field as a whole.

**Comment by flaviocopes** [comment]
Sure, merged

**Comment by OleVik** [comment]
Note to self: range.html.twig is not overriding anything, it is a field that does not currently exist in the Admin-plugin.

**Comment by OleVik** [comment]
Solved with [v1.0.3-RC.1](https://github.com/OleVik/grav-plugin-adminidenticons/releases/tag/v1.0.3-RC1), releasing when Admin hits v1.3.2.
